src/org/zaproxy/zap/extension/bugtracker/resources/help/contents/bugtracker.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>Bug Tracker</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Bug Tracker</H1>
+<p>
+The Bug Tracker add-on can help the user raise the Alerts that they recieve in the form of an issue or a bug on a Bug Tracker (like Bugzilla, Github,etc) while scanning a web-app.</p><br>
```

Comment:
There's a typo "recieve".
Do we really need the break lines after the paragraphs?